src/xenia/kernel/util/object_table.cc

```diff
@@ -21,6 +21,23 @@ namespace xe {
 namespace kernel {
 namespace util {
 
+namespace {
+#if defined(XE_PLATFORM_LINUX)
+#include <cxxabi.h>
+#include <memory>
+std::string demangle(const char* mangled_name) {
```

Comment:
As demangling may be useful in other places, it'd possibly be nice to have this somewhere in `xenia/base`.

Reply:
Done